nnunet/README.md

```diff
@@ -149,7 +141,23 @@ python -m monai.apps.nnunet nnUNetV2Runner predict_ensemble_postprocessing --inp
 ## [component] post-processing only
 python -m monai.apps.nnunet nnUNetV2Runner predict_ensemble_postprocessing --input_config "./input.yaml" \
 	--run_predict false --run_ensemble false
+```
+
+For utilizing PyTorch DDP in multi-GPU training, the subsequent command is offered to facilitate the training of a singlular model on a specific fold:
 
+```bash
+## [component] multi-gpu training for a single model
+python -m monai.apps.nnunet nnUNetV2Runner train_single_model --input_config "./input.yaml" \
+    --config "3d_fullres" \
+    --fold 0 \
+    --gpu_id 0,1
+```
+
+We offer an alternative API for constructing datasets from [MSD challenge](http://medicaldecathlon.com/) to meet requirements of nnU-Net, as reference in the provided [link](https://github.com/MIC-DKFZ/nnUNet/blob/master/documentation/dataset_format.md#how-to-use-decathlon-datasets).
+
+```bash
+## [component] converting msd datasets
+python -m monai.apps.nnunet nnUNetV2Runner convert_msd_dataset --input_config "./input.yaml" --data_dir "/workspace/data/Task09_Spleen"
 ```
```

Comment:
there's a testing issue, could you please have a look? https://github.com/Project-MONAI/MONAI/issues/6496

Reply:
@wyli I submitted a PR in https://github.com/Project-MONAI/MONAI/pull/6499. Thanks!